tools/pipeline_perf_test/load_generator/loadgen-manifest.yaml

```diff
@@ -24,8 +24,8 @@ spec:
         - "{{DURATION}}"  # This will be replaced by the orchestrator
         resources:
           requests:
-            cpu: 100m
-            memory: 100Mi
+            cpu: "2"
+            memory: 1000Mi
           limits:
-            cpu: 500m
-            memory: 500Mi
+            cpu: "4"
```

Comment:
In general we should avoid cpu limits, and ensure we have cpu requests set at <some to be determined rational baseline> value(s) on all the containers. Quick read on why these aren't great in most cases:

https://home.robusta.dev/blog/stop-using-cpu-limits

Reply:
thanks! I added this so that the load-generator was able to keep the Collector busy! 
Lets find a better way to handle this!

Reply:
Agree! I remember at my old company we had come to the conclusion that limits must equal requests otherwise good luck understanding your performance.